content/blog/2022-06-07-using-gcp-remotes-in-dvc.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Syncing Data to GCP Storage Buckets
+date: 2022-06-07
+description: >
+  We're going to set up an GCP storage bucket remote in a DVC project.
+descriptionLong: >
+  Setting up a remote to make data versioning easier with DVC is a common need
+  so we're going to go through a tutorial for doing this with GCP.
+picture: 2022-06-07/dvc-gcp.png
+pictureComment: Using GCP Remotes in DVC
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/preventing-stale-models-in-production/1137
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - DVC Remotes
+  - Collaboration
+---
+
+When you’re working on a data science project that has huge datasets, it’s
+common to store them in cloud storage. You’ll also be working with different
+versions of the same datasets to train a model, so it’s crucial to have a tool
+that enables you to do this quickly and easily. That’s why we’re going to do a
+quick walkthrough of how to set up a remote in a GCP storage bucket and handle
+data versioning with [DVC](https://dvc.org/doc).
+
+We’ll start by creating a new storage bucket in our GCP account, then we’ll show
+how you can add DVC to your project, and finally, we’ll make updates to the
+dataset with DVC commands. We’ll be working with
+[this repo](https://github.com/iterative/stale-model-example) if you want an
+example to play with. By the time you finish, you should be able to create this
+setup for any machine learning project using an GCP remote.
+
+## Set up a GCP storage bucket
+
+Make sure that you already have a
+[GCP account](https://console.cloud.google.com). You’ll need a valid credit card
+to create a new account. Once you’re logged into your account, you should see a
```

Comment:
It might be worth to mention that GCP has a [free tier](https://cloud.google.com/free/docs/gcp-free-tier) that can be used to play around with GCP without being billed